A few tests are failing on Windows

I am getting the following when running all the tests:

```
0
--- FAIL: TestMain_OsArgs (0.24s)
    --- FAIL: TestMain_OsArgs/0-Install_Test (0.24s)
panic: runtime error: index out of range [0] with length 0 [recovered]
        panic: runtime error: index out of range [0] with length 0

goroutine 20 [running]:
testing.tRunner.func1.2({0xaf5060, 0xc0004ce180})
        C:/Program Files/Go/src/testing/testing.go:1209 +0x24e
testing.tRunner.func1()
        C:/Program Files/Go/src/testing/testing.go:1212 +0x218
panic({0xaf5060, 0xc0004ce180})
        C:/Program Files/Go/src/runtime/panic.go:1038 +0x215
github.com/everettraven/packageless/utils.(*Utility).ImageExists(0x0, {0xc00001e1e0, 0x19}, {0xc050b0, 0xc000538000})
        C:/Users/kasto/Projects/packageless/utils/docker.go:52 +0x165
github.com/everettraven/packageless/subcommands.(*InstallCommand).Run(0xc000378480)
        C:/Users/kasto/Projects/packageless/subcommands/install_sc.go:150 +0x67e
github.com/everettraven/packageless/subcommands.SubCommand({0xc000392cd0, 0x2, 0x2}, {0xc0003f5e58, 0x6, 0x39})      
        C:/Users/kasto/Projects/packageless/subcommands/subcommand.go:31 +0x1b3
github.com/everettraven/packageless.wrappedMain()
        C:/Users/kasto/Projects/packageless/main.go:72 +0xafb
github.com/everettraven/packageless.TestMain_OsArgs.func1(0xc0003da000)
        C:/Users/kasto/Projects/packageless/main_test.go:185 +0xbf
testing.tRunner(0xc0003da000, 0xc00011ce88)
        C:/Program Files/Go/src/testing/testing.go:1259 +0x102
created by testing.(*T).Run
        C:/Program Files/Go/src/testing/testing.go:1306 +0x35a
FAIL    github.com/everettraven/packageless     0.531s
ok      github.com/everettraven/packageless/subcommands 0.291s  coverage: 91.7% of statements
ImagePullthe input device is not a TTY.  If you are using mintty, try prefixing the command with 'winpty'
the input device is not a TTY.  If you are using mintty, try prefixing the command with 'winpty'
{"status":"Pulling from bpalmer/alpine-base-ssh","id":"latest"}
{"status":"Digest: sha256:55cc4cb96fb5ab814eaa33fdbf388a5bdfd4d6d1fea61e46f47561f34bd71087"}
{"status":"Status: Image is up to date for bpalmer/alpine-base-ssh:latest"}
--- FAIL: TestDocker_Integration (0.87s)
panic: runtime error: index out of range [0] with length 0 [recovered]
        panic: runtime error: index out of range [0] with length 0

goroutine 37 [running]:
testing.tRunner.func1.2({0x82e100, 0xc0000220d8})
        C:/Program Files/Go/src/testing/testing.go:1209 +0x24e
testing.tRunner.func1()
        C:/Program Files/Go/src/testing/testing.go:1212 +0x218
panic({0x82e100, 0xc0000220d8})
        C:/Program Files/Go/src/runtime/panic.go:1038 +0x215
github.com/everettraven/packageless/utils.(*Utility).ImageExists(0xc000109d60, {0xc00008dd08, 0x1e}, {0x93d2b0, 0xc00042c200})
        C:/Users/kasto/Projects/packageless/utils/docker.go:52 +0x19f
github.com/everettraven/packageless/utils.TestDocker_Integration(0xc0004164e0)
        C:/Users/kasto/Projects/packageless/utils/docker_test.go:621 +0x1d4
testing.tRunner(0xc0004164e0, 0x889698)
        C:/Program Files/Go/src/testing/testing.go:1259 +0x102
created by testing.(*T).Run
        C:/Program Files/Go/src/testing/testing.go:1306 +0x35a
FAIL    github.com/everettraven/packageless/utils       3.261s
FAIL
```

**To Reproduce**
Steps to reproduce the behavior:
1. Run `go test --coverprofile=coverage.out ./...` at the project root

**Expected behavior**
All tests should pass.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome

--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,4 @@ python
 packageless.exe
 macros.doskey
 packageless
+*.out

--- a/utils/docker.go
+++ b/utils/docker.go
@@ -49,6 +49,12 @@ func (u *Utility) ImageExists(imageID string, cli Client) (bool, error) {
 
 	//Loop through all the images and check if a match is found
 	for _, image := range images {
+
+		// If RepoTags returned isnt populated then skip to the next image
+		if len(image.RepoTags) < 1 {
+			continue
+		}
+
 		if image.RepoTags[0] == imageID {
 			return true, nil
 		}

--- a/utils/docker_test.go
+++ b/utils/docker_test.go
@@ -166,6 +166,44 @@ func TestImageExistError(t *testing.T) {
 	}
 }
 
+//Test ImageExists function when RepoTags is not populated
+func TestImageExistContinueOnEmptyRepoTag(t *testing.T) {
+	//Create the Mock Docker Client
+	dm := NewDockMock()
+
+	//Set the image for testing
+	img := "image:faketag"
+
+	//Set the return images array in the Mock Docker Client
+	dm.ILRet = []types.ImageSummary{
+		// First one should be an empty RepoTags
+		{
+			RepoTags: []string{},
+		},
+		// When image loop continues it should find this one.
+		{
+			RepoTags: []string{"image:faketag"},
+		},
+	}
+
+	//Create a new utility
+	util := NewUtility()
+
+	//Check for the image
+	exists, err := util.ImageExists(img, dm)
+
+	//If an error occurs, the test fails
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	//The image should exist in this case
+	if !exists {
+		t.Fatal("ImageExists: Image should exist, but it does not.")
+	}
+
+}
+
 //Test CreateContainer Function
 func TestCreateContainer(t *testing.T) {
 	//Create the Mock Docker Client

